Packs/AWS-Enrichment-Remediation/Scripts/AWSAccountHierarchy/AWSAccountHierarchy.py

```diff
@@ -0,0 +1,151 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from typing import Dict, Any
+import traceback
+
+
+""" STANDALONE FUNCTION """
+
+
+def lookup(parent_obj: str, level: int, instance_to_use: str) -> tuple[str, dict]:
```

Comment:
Why is the **instance_to_use** mandatory?

Reply:
So in the case that there are multiple `AWS - Organizations` integration instances configured and you don't know which will return data, I store the instance that returned data: https://github.com/PaloAltoNetworks/xpanse-content/blob/9f64c8eb2d32d00e116ba543ce86b1f4dd81281e/Packs/AWS-Enrichment-Remediation/Scripts/AWSAccountHierarchy/AWSAccountHierarchy.py#L96C12-L96C12

I then use this for all other `aws-org` commands so that it only needs to try the command on the right instance instead of multiple.  Make sense?

Did the same in this script https://github.com/PaloAltoNetworks/xpanse-content/blob/9f64c8eb2d32d00e116ba543ce86b1f4dd81281e/Packs/AWS-Enrichment-Remediation/Scripts/AWSRecreateSG/AWSRecreateSG.py#L287